test/parallel/test-crypto.js

```diff
@@ -54,19 +54,18 @@ assert.throws(function() {
 
 
 function assertSorted(list) {
-  // Array#sort() modifies the list in place so make a copy.
-  const sorted = list.slice().sort();
-  assert.deepStrictEqual(list, sorted);
+  // list is sorted and contains no duplicates
+  assert(list.every(
+    (val, index, array) => index === array.length - 1 || val < array[index + 1]
+  ));
 }
 
 // Assume that we have at least AES-128-CBC.
-assert.notStrictEqual(0, crypto.getCiphers().length);
```

Comment:
> Without this, getCiphers() could return [], and the test would pass, because the empty set is sorted and unique.

The test will fail because the very next line requires that the array includes `aes-128-cbc` as an element. So it can't be length 0. So the zero-length check seems redundant to me.

Although looking more closely, I suppose it's possible that `getCiphers()` would return different arrays each time called. (That would be a bug, but that's the point of tests.) So I should probably assign the result to a variable before all the asserts. Maybe even throw in another call to `getCiphers()` at the end and confirm that the array it returns is identical to the one it returned in the first place.